.gitignore

```diff
@@ -64,6 +64,7 @@ src/Application/test/calculator_test
 
 # ignore .vscode
 .vscode/
+.vscode/*
```

Comment:
You also need to stop tracking the files, otherwise they will continue to show up

Reply:
Use this SO answer as reference https://superuser.com/a/1442728